projects/shr-mediator/src/common/services/fhir.service.ts

```diff
@@ -101,17 +118,29 @@ export class FhirService {
     options?: AxiosRequestConfig<any>,
     retryLimit = 2,
     timeout = 30000,
+    path?: string
```

Comment:
```suggestion
    path?: string = ''
```
Let's rather do this.